chart/templates/_commonChecks.tpl

```diff
@@ -161,6 +216,57 @@ Return customer values to use in preflights and support-bundle
     value: {{ include "carto.postgresql.databaseName" . }}
   - name: WORKSPACE_POSTGRES_USER
     value: {{ include "carto.postgresql.user" . }}
+  - name: WORKSPACE_TENANT_ID
+    value: {{ .Values.cartoConfigValues.selfHostedTenantId | quote }}
+  {{- if not .Values.commonBackendServiceAccount.enableGCPWorkloadIdentity }}
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: {{ include "carto.google.secretMountAbsolutePath" . }}
+  {{- end }}
+  - name: WORKSPACE_THUMBNAILS_BUCKET
+    value: {{ .Values.appConfigValues.workspaceThumbnailsBucket | quote }}
+  - name: WORKSPACE_IMPORTS_BUCKET
+    value: {{ .Values.appConfigValues.workspaceImportsBucket | quote }}
+  - name: WORKSPACE_THUMBNAILS_PROVIDER
+    value: {{ .Values.appConfigValues.storageProvider | quote }}
+  - name: WORKSPACE_IMPORTS_PROVIDER
+    value: {{ .Values.appConfigValues.storageProvider | quote }}
+  - name: WORKSPACE_THUMBNAILS_PUBLIC
+    value: {{ .Values.appConfigValues.workspaceThumbnailsPublic | quote }}
+  - name: WORKSPACE_IMPORTS_PUBLIC
+    value: {{ .Values.appConfigValues.workspaceImportsPublic | quote }}
```

Comment:
Why you use that value but then we have this: https://github.com/CartoDB/carto-selfhosted-helm/blob/main/manifests/kots-helm.yaml#L91 ? :thinking:

Reply:
This is at HELM chart level, not at Kots level 🙇🏼‍♂️